protocol/syncer.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	maxEnqueueSize = 50
-	popTimeout     = time.Second * 10
+	popTimeout     = 20 * time.Second
```

Comment:
How did you come up with this number? `20s`?

Reply:
When you want to grow something you usually double it :) I remember we mentioned this on the meeting, but it really doesn't matter how long this is.

Reply:
@brkomir We talked about decreasing this to the previous value.